src/sentry/runner/commands/cleanup.py

```diff
@@ -270,7 +270,9 @@ def is_filtered(model: type[Model]) -> bool:
                     order_by=order_by,
                 )
 
-                for chunk in q.iterator(chunk_size=100, use_range_wrapper=use_range_wrapper):
+                for chunk in q.iterator(
+                    chunk_size=100, batch_size=10000, use_range_wrapper=use_range_wrapper
```

Comment:
Should we instead change the default value?
https://github.com/getsentry/sentry/blob/82de8bb53a7df7836d0c5ded0ced2eb055cddb52/src/sentry/db/deletion.py#L85

Reply:
That might make sense, I can do it once we see that it's helpful, and when I merge https://github.com/getsentry/sentry/pull/100642 to enable the range wrapper for all deletions